docs/prod-host-setup.md

```diff
@@ -220,14 +220,24 @@ See more details [here](https://www.kernel.org/doc/html/latest/admin-guide/hw-vu
 This will mitigate variants of Spectre side-channel issues such as
 Speculative Store Bypass and SpectreNG.
 
-It can be enabled by adding the following Linux kernel boot parameter:
+On x86_64 systems, it can be enabled by adding the following Linux kernel boot
+parameter:
 
 ```console
 spec_store_bypass_disable=seccomp
 ```
 
 which will apply SSB if seccomp is enabled by Firecracker.
 
+On aarch64 systems, it is done in the jailer [using the `prctl` interface][3].
```

Comment:
```suggestion
On aarch64 systems, it is enabled by Firecracker [using the `prctl` interface][3].
```

Reply:
Please also update the commit message, which still mentions the jailer

Reply:
Done.